open-craft/openedx-deployment
# Updates analytics deployment docs to use terraform

Updates the analytics deployment documentation to:

* Replace the manual resource provisioning with our latest approach developed in [terraform-scripts](https://github.com/open-craft/terraform-scripts/blob/main/modules/services/analytics).
* Updates the OAuth2 documentation to use the ansible variables added for Django Auth Toolkit applications.
* Adds details about running Open edX Analytics on non-US AWS regions and using S3 SigV4.

**Jira ticket**

* [FAL-1720](https://tasks.opencraft.com/browse/FAL-1720) (Opencraft access only)

**Testing instructions**

Other than fully deploying a new analytics instance using these updated instructions, I think the only way to really test and verify these changes is to check them against our known working UBC client files and docs.

These updated docs can be used for our next new deployment and for the scheduled upgrades, and any fixes applied if issues are noticed.

**Reviewer**
